### PR TITLE
more alt text aria stuff, and footer changes

### PR DESCRIPTION
**Pull request by UcDust**
Hi @spelkey-ucd, one note on the footer, Kimmy wanted it to match the UC pattern like the footer at https://www.ucdavis.edu/. In addition to the links, they also have a `Last update: some date` label. I hard coded that, is there another preferred method? Looking at the footer customization in the admin portal, I didn't see a straight text element where you add the links. Anyway, just thought I'd call that out, maybe it's ok.

### CODE REVIEW

File: views/template-partials/footer.twig [hunk 1]
```diff
@@ -26,10 +26,14 @@
               <a href="{{ item.link }}" {{ item.is_target_blank ? 'target="_blank"' }}>{{ item.title }}</a>
             </li>
           {% endfor %}
+          <li>
+              Last update: August 16, 2022
+          </li>
```
Comment (spelkey-ucd) [inline views/template-partials/footer.twig:31]: remove this. we already have an option to add build info.

File: views/template-partials/footer.twig [hunk 2]
```diff
@@ -26,10 +26,14 @@
...
         {% endif %}
       </ul>
       <p>Copyright © The Regents of the University of California, Davis campus. All rights reserved.</p>
     </div>
+    <div class="u-align--center u-space-mt--large">This site is officially grown in <a href="https://sitefarm.ucdavis.edu">SiteFarm</a>.</div>
```
Comment (spelkey-ucd) [inline views/template-partials/footer.twig:36]: this is not true...

**Review by spelkey-ucd — CHANGES_REQUESTED** [review]

File: views/template-partials/footer.twig (hunk 1 above)
Comment (UcDust) [reply views/template-partials/footer.twig]: Ok, done.

File: views/template-partials/footer.twig (hunk 2 above)
Comment (UcDust) [reply views/template-partials/footer.twig]: Kimmy had said everything above the Aggie's logo is ours, but the content below should match the other UC sites. Sorry about that. I've removed that line.

**Pull request merged**